modules/olm-arch-olm-operator.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-arch.adoc
+
+[id="olm-arch-olm-operator_{context}"]
+= OLM Operator
+
+The OLM Operator is responsible for deploying applications defined by CSV
+resources after the required resources specified in the CSV are present in the
+cluster.
+
+The OLM Operator is not concerned with the creation of the required resources;
+users can choose to manually create these resources using the CLI, or users can
+choose to create these resources using the Catalog Operator. This separation of
+concerns allows users incremental buy-in in terms of how much of the OLM
+framework they choose to leverage for their application.
+
+.OLM Operator workflow
+* Watches for ClusterServiceVersion (CSVs) in a namespace and checks that
+requirements are met. If so, runs the install strategy for the CSV.
```

Comment:
Similar to a previous comment, I think this could be helped by writing out the complete sentence.